### ISSUE TITLE
Mods/Administrators fields are not saved

### ISSUE THREAD

**Issue by Jonathan-Rosewood** [TorchAPI/Torch#64]
**Torch Version:** 1.1.213.390 **SE Version:** 1.182.103

### Expected Behavior
Saving the values of these fields.

### Observed Behavior
Field values are not saved.

### Steps to Reproduce
1. Launch the Torch application
2. Enter the data in the "Administrators" and "Mods" fields.
3. Click the "Save Config" button.
4. Close the Torch application

### Other Information
[screenshot before relaunch](http://prntscr.com/g3jimq)

**Comment by kurazarrh** [comment]
Are you running Torch.exe as an administrator? There's a well-documented bug regarding Torch's inability to save its configuration unless it's being run with elevated privileges.

**Comment by Jonathan-Rosewood** [comment]
At the expense of privilege did not try. I'll do it.
But I can say one more interesting thing.
I tried to manually add mods to the config, but they are deleted when the application is launched

Used mod item syntax:
```
<ModItem>
  <Name>ModID.sbm</Name>
  <PublishedFileId>ModId</PublishedFileId>
</ModItem>
```

**Comment by Jonathan-Rosewood** [comment]
Tried to run with administrator rights - unsuccessfully.

Actions:
- Launched application with administrator privileges
- Added value "652337022" in the field "Mods"
- Clicked the button "Save Config"
- Closed application
- Launched application with administrator privileges

**Comment by finne132** [comment]
I can confirm that this is an issue for me as well and that admin rights don't appear to have any effect. I can also confirm that the server config file is over-written on server launch, so if you manually add mods they are wiped out on server start.

Will continue to try to find a workaround and post any updates here

**Comment by haximodor92** [comment]
I confirm the same situation.

**Comment by finne132** [comment]
This thread appears to be a duplicate of https://github.com/TorchAPI/Torch/issues/49 

I tagged the creator in that thread and mentioned this is still an issue. Also in that thread there's a brief mention of being able to launch w/mods using a manual workaround. I've asked the poster to provide that info as well if possible :) !

**Comment by Jimmacle** [comment]
Fixed 82815f66e5384b8e8427d50720e23aebd3e10da0
